pages/_app.tsx

```diff
@@ -29,10 +66,25 @@ const Agera = ({ Component, pageProps }: AppProps) => {
 
   return (
     <ThemeProvider theme={theme}>
-      <CssBaseLine />
+      <CssBaseline />
 
       <Layout>
         <ErrorBoundary>
+          {loading ? (
+            <LinearProgress
+              color="secondary"
+              className="nav-loading"
```

Comment:
`nav-loading` is global class?

Reply:
It was a global class name but can be removed because corresponding styles are specified with `sx` attributes. I'll remove this class

Reply:
Updated in [70c9991](https://github.com/nervosnet/godwoken-explorer-ui/pull/129/commits/70c9991b4dd27c6dfc38925de18cf9c08416d17e)